state-chain/chains/src/sol/api.rs

```diff
@@ -767,3 +767,30 @@ pub enum AltConsensusResult<T> {
 	ValidConsensusAlts(T),
 	AltsInvalidNoConsensus,
```

Comment:
Naming is inconsistent, we end one with Alts, and start one with Alts.... we don't even need to include "Alts" in the variants, it's on the type.
```suggestion
	ValidConsensus(T),
	InvalidNoConsensus,
```